app/js/lib/foxbox/settings.js

```diff
@@ -203,4 +187,33 @@ export default class Settings extends EventDispatcher {
   get apiVersion() {
     return API_VERSION;
   }
+
+  clear() {
+    return new Promise((resolve) => {
+      this.session = null;
+      // @todo Remove only the items set here.
```

Comment:
Can we fix this todo? Now that we have the list of properties in an array, we can iterate through them and selectively delete them while preserving the others.